src/store/wizardSlice.ts

```diff
@@ -117,6 +135,22 @@ export const selectActivationKey = (state: RootState) => {
   return state.wizard.registration.activationKey;
 };
 
+export const selectProfile = (state: RootState) => {
+  return state.wizard.profile;
+};
+
+export const selectOscapProfileId = (state: RootState) => {
+  return state.wizard.openScap.profileId;
+};
+
+export const selectOscapProfileName = (state: RootState) => {
+  return state.wizard.openScap.profileName;
+};
+
+export const selectOscapProfileDescription = (state: RootState) => {
+  return state.wizard.openScap.profileDescription;
+};
+
```

Comment:
This would just get small update in the structure:
- state.wizard.profile -> state.wizard.openScap.profile
- state.wizard.openScap.profileId -> state.wizard.openScap.profileInformation.profileId